docs/getting-started/db.md

```diff
@@ -46,10 +46,23 @@ It's currently used only for `articles` and `users`.
 
 It can also be used for `rails db:seed` and `rails db:reset`.
 
+## Default Admin User
+
+Seed data creates a handful of regular users, and a single admin user that can
+be used to log into the application with the Email login option:
+
+```
+email: admin@forem.local
+password: admin
```

Comment:
Isn't the password 'password' not admin?

Reply:
It is indeed. Want to make the doc update PR? 😃